pkg/controller/job-controller.go

```diff
@@ -85,7 +79,28 @@ func (c *JobController) processNextJobWorkItem() bool {
 		}
 		klog.V(2).Infof("Key from workqueue: %s", key)
```

Comment:
maybe not really worth a log line? could be helpfull for debug, not so much on production